src/main/java/justin/Ui.java

```diff
@@ -0,0 +1,153 @@
+package justin;
+
+import justin.task.Task;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a class that is responsible for
+ * sending outputs to users.
```

Comment:
I think could be more representative of Ui, as it sounds better to say it handles what is displayed on the user interface?